app/views/talks/index.html.slim

```diff
@@ -10,7 +10,18 @@
       .col-xs-12.col-sm-5.col-sm-offset-1
         = render 'form'
       .col-xs-12.faq.col-sm-6
-        = markdown t('.faq')
+        h2 Je suis un newbie, et j'ai peeeeuuur de présenter
```

Comment:
Before:

![Screenshot_2021-11-06_22-45-06](https://user-images.githubusercontent.com/1866809/140624748-eda3e1dc-8ad4-4fb9-a6d5-028c169c0d00.png)

After:

![Screenshot_2021-11-06_22-45-38](https://user-images.githubusercontent.com/1866809/140624752-908a21ec-f4d2-4b07-8896-49ea08b5deab.png)


(it is the same)